src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -91,7 +91,7 @@ public without sharing class UTIL_CustomSettingsFacade {
      * Private methods that will configure the default org-level custom settings.
      **/
     private static void configSettings(Hierarchy_Settings__c hs) {
-		hs.Account_Processor__c = 'Administrative';
+		hs.Account_Processor__c = UTIL_Describe.getRecTypesMapByDevName('Account').get('Administrative');
```

Comment:
This is looking at the Developer Name, which is the same even if the label gets changed. Also, configSettings is only called if no Hierarchy Settings exist. The assumption is that this is done only once, after installation. If our Administrative record type gets deleted, hopefully they will select another one (maybe one they created) in the Account Processor field in settings.